### DISCUSSION TITLE
Portals to Beautiful Futures - report from Omidyar and Future Architects

### DISCUSSION THREAD

**Discussion by rufuspollock** [life-itself/community#108] [General]
[Portals-to-Beautiful-Futures-2021.pdf](https://github.com/life-itself/community/files/7752192/Portals-to-Beautiful-Futures-2021.pdf)

**Comment by baouroux** [comment]
Wow, an impressive document and resource....Fascinating to learn of the initiatives GoFA and Omidyar.

A more historical, theoretical and analytical paper can be found here, from the prolific Anthony Judge: "[Consciously Self-reflexive Global Initiatives Renaissance zones, complex adaptive systems, and third order organizations](https://www.laetusinpraesens.org/docs00s/thirdord.php)"

Encompasses organisational and societal futures...not entirely sure where to quote [Anthony Judge](https://www.laetusinpraesens.org/) in this community portal, since he's covered over the past 30+ years so much writing on futures and development!
